src/ruby/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'rdoc', '~>4.0'
+gem 'rspec', '~> 3.0'
```

Comment:
I didn't know that. Thanks! Does bundler still work after the dependencies are moved to gemspec?